app/managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "

```

Comment:
It is common to use triple quotes for multilne queries

Reply:
Ok, but then there is a need to change the videos on the platform, because the lecturer there doesn't do that...